libdrgn/drgn.h

```diff
@@ -930,6 +930,73 @@ struct drgn_error *drgn_program_find_symbols_by_address(struct drgn_program *pro
 							struct drgn_symbol ***syms_ret,
 							size_t *count_ret);
 
+/** Flags for @ref drgn_symbol_find_fn() */
+enum drgn_find_symbol_flags {
+	/** Find symbols whose name matches the name argument */
+	DRGN_FIND_SYMBOL_NAME = 1 << 0,
+	/** Find symbols whose address matches the addr argument */
+	DRGN_FIND_SYMBOL_ADDR = 1 << 1,
+	/** Find only one symbol */
+	DRGN_FIND_SYMBOL_ONE = 1 << 2,
+};
+
+/** Result builder for @ref drgn_symbol_find_fn() */
+struct drgn_symbol_result_builder;
+
+/**
+ * Add or set the return value for a symbol search
+ *
+ * Symbol finders should call this with each symbol search result. If the symbol
+ * search was @ref DRGN_FIND_SYM_ONE, then only the most recent symbol added to
```

Comment:
`s/DRGN_FIND_SYM_ONE/DRGN_FIND_SYMBOL_ONE/`

Reply:
You know, maybe this should be my push to use a better find-replace option than what I normally do. I use clangd's rename option all the time for renaming struct fields, etc. It's great because it is type-aware, so it can avoid doing renames on other colliding entities. But the downside is that it doesn't bother with comments.